minimedia.cpp

```diff
@@ -50,6 +52,10 @@ main(int, char**)
     sp<ProcessState> proc(ProcessState::self());
     sp<IServiceManager> sm = defaultServiceManager();
 
+    // Disable things which break hybris once and for all.
+    property_set("persist.camera.shutter.disable", "1");
+    property_set("camera.fifo.disable", "1");
```

Comment:
If this happens early enough in boot for all things to work, more than happy to LGTM this!

Reply:
yep it happens right before camera is loaded ;).

Reply:
My only reservation about this is that it's so obvious that I'd assume it would have been done in the first place if it didn't cause some problem.